### DISCUSSION TITLE
Moving to a new database

### DISCUSSION THREAD

**Discussion by PaulMcF1987** [Budibase/budibase#6760] [Help]
Currently, the database that I am using is a MYSQL Docker hosted DB. On my old server the databases were locked down to internal apps only and docker was seen as an external app.
Now I have changed server, I am able to use the DB that is hosted on the server, which I feel would be a safer option.

I have copied the databases over.

Rather than change the DB in the app, I opted to copy the app and try the new DB on the copy first.

I have found that when I import the data from the old table, the `multiselect` column changes to a `text` column and imports in the form

`{"0":"Service1","1":"Service1"...} rather than showing each of the options selected. when  I click Edit to and scroll to the column it is populated with `Null`

How do I import this correctly?

I have moved from a Docker hosted MYSQL DB to my server hosted MariaDB
The server is an Ubuntu 20.04

--------------------------------------------------------------------------------------------------------------------------------------------

Also, when importing the app from an export of the original app... When I try to

**Comment by melohagan** [comment]
Hi @PaulMcF1987 

It may be that when importing data columns are always defaulted to text, and the Budibase type is not saved. 

It might help to see some screenshots as well just to make sure I understand you correctly.

**Reply by PaulMcF1987** [reply]
@melohagan I have went through all of the records and added these areas manually.

Although, I have noticed that when I try and add the details to my customers table, that MariaDB doesnt see to save details as `JSON` and instead replaces with `LongText` which means I can not use the {{ pluck table.column 'primaryDisplay'}}
Is there an alternative that I can use insteadso that it is displayed a bit better
Right now it is displayed `["Selection1","Selection2","Selection3"]`
